src/shared/flutter/daemon_interfaces.ts

```diff
@@ -78,3 +78,8 @@ export interface ShowMessage {
 export interface SupportedPlatformsResponse {
 	platforms: PlatformType[];
 }
+
+export interface ServeDevToolsResponse {
+	host: string;
+	port: string;
```

Comment:
Based on the code above, this maybe should include `| null` (or `| undefined`)?

Reply:
I think truthiness check makes sense for this case, so in that case I think you mean that we don't need null/undefined here?

Reply:
If the value _could_ be `null` or `undefined`, we should include it here (otherwise it looks like those are not valid values and we may accidentally write code that assumes it's never `null`/`undefined` in the future).

I'm assuming it can be null because of the code above (`result.host != null && result.port != null`), but I'm not certain (this depends on what the internal daemon can emit).

Reply:
Oops I misunderstood what code we were looking at. It looks like these fields could both be null.